specs/123-mobile-responsive-tables-tabs/quickstart.md

```
@@ -0,0 +1,114 @@
+1. Open Chrome/Firefox DevTools (F12)
+2. Toggle device toolbar (Ctrl+Shift+M / Cmd+Shift+M)
+3. Select a mobile preset:
+   - **iPhone SE**: 375px (minimum target)
+   - **iPhone 14**: 390px
+   - **Pixel 7**: 412px
+4. Navigate to any table or tabbed page
+
+### Key Pages to Test
+
+| Page | URL | What to Verify |
+|------|-----|----------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)